plugins/usbdmx/ShowJockeyFactory.cpp

```diff
@@ -32,6 +32,9 @@ namespace usbdmx {
 
 using ola::usb::LibUsbAdaptor;
 
+const char ShowJockeyFactory::EXPECTED_MANUFACTURER[] = "Showjockey Co.,Ltd";
+const char ShowJockeyFactory::EXPECTED_PRODUCT[] =
+                                            "Showjockey Co.,Ltd.USB TO DMX51";
```

Comment:
Can you just indent this 4 please. Is it really DMX51 (did they run out of space for the 2)?